src/__tests__/lib/containers/entity/page/title_bar/EntityPageTitleBar.test.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import EntityPageTitleBar, {
+  EntityPageTitleBarProps,
+} from '../../../../../../lib/containers/entity/page/title_bar/EntityPageTitleBar'
+import { render, screen } from '@testing-library/react'
+import { createWrapper } from '../../../../../../lib/testutils/TestingLibraryUtils'
+import mockFileEntity from '../../../../../../mocks/entity/mockFileEntity'
+import { EntityActionMenuProps } from '../../../../../../lib/containers/entity/page/action_menu/EntityActionMenu'
+import {
+  EntityBundle,
+  EntityType,
+} from '../../../../../../lib/utils/synapseTypes'
+import { rest, server } from '../../../../../../mocks/msw/server'
+import {
+  BackendDestinationEnum,
+  getEndpoint,
+} from '../../../../../../lib/utils/functions/getEndpoint'
+import { ENTITY_BUNDLE_V2 } from '../../../../../../lib/utils/APIConstants'
+import * as FavoriteButtonModule from '../../../../../../lib/containers/favorites/FavoriteButton'
+import * as EntityActionMenuModule from '../../../../../../lib/containers/entity/page/action_menu/EntityActionMenu'
+import * as TitleBarPropertiesModule from '../../../../../../lib/containers/entity/page/title_bar/TitleBarProperties'
+import * as TitleBarVersionInfoModule from '../../../../../../lib/containers/entity/page/title_bar/EntityTitleBarVersionInfo'
+import failOnConsoleError from 'jest-fail-on-console'
+
+const TITLE_BAR_PROPERTIES_TEST_ID = 'title-bar-properties'
+const TITLE_BAR_VERSION_INFO_TEST_ID = 'title-bar-version-info'
+const ENTITY_ACTION_MENU_TEST_ID = 'entity-action-menu'
+const FAVORITE_BUTTON_TEST_ID = 'favorite-button'
+
+jest
+  .spyOn(TitleBarPropertiesModule, 'default')
+  .mockImplementation(() => (
+    <div data-testid={TITLE_BAR_PROPERTIES_TEST_ID}></div>
+  ))
+
+jest
+  .spyOn(TitleBarVersionInfoModule, 'EntityTitleBarVersionInfo')
+  .mockImplementation(() => (
+    <div data-testid={TITLE_BAR_VERSION_INFO_TEST_ID}></div>
+  ))
+
+jest
+  .spyOn(EntityActionMenuModule, 'default')
+  .mockImplementation(() => (
+    <div data-testid={ENTITY_ACTION_MENU_TEST_ID}></div>
+  ))
+
+jest
+  .spyOn(FavoriteButtonModule, 'default')
+  .mockImplementation(() => <span data-testid={FAVORITE_BUTTON_TEST_ID}></span>)
```

Comment:
The complexity in all of these components are individually tested, so just mock them all for this test.